src/org/ods/component/ScanWithSnykStage.groovy

```diff
@@ -20,6 +20,11 @@ class ScanWithSnykStage extends Stage {
     if (!config.buildFile) {
       config.buildFile = 'build.gradle'
     }
+    if (!config.severityThreshold) {
+      // low is the default, other possible values are medium and high.
```

Comment:
It would be nice to convert this comment to an error message that is given to the user when the given value is not in the allowed list. That way we serve both developer (reading the code) and user (calling the stage).